mne/io/ctf/tests/test_ctf.py

```diff
@@ -211,4 +212,13 @@ def test_read_spm_ctf():
     assert_equal(extras['n_samp'], raw.n_times)
     assert_false(extras['n_samp'] == extras['n_samp_tot'])
 
+    # Test that LPA, nasion and RPA are correct.
+    coord_frames = np.array([d['coord_frame'] for d in raw.info['dig']])
+    np.all(coord_frames == FIFF.FIFFV_COORD_HEAD)
+    cardinals = {d['ident']: d['r'] for d in raw.info['dig']}
+    assert_true(cardinals[1][0] < cardinals[2][0] < cardinals[3][0])  # x coord
+    assert_true(cardinals[1][1] < cardinals[2][1])  # y coord
+    assert_true(cardinals[3][1] < cardinals[2][1])  # y coord
```

Comment:
and all of the z coords should be `assert_allclose(zcoords, 0, atol=1e-6)` right?